python/lsst/ts/ofc/scripts/generate_alpha_values.py

```diff
@@ -0,0 +1,271 @@
+# This file is part of ts_ofc.
+#
+# Developed for Vera Rubin Observatory.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import argparse
+import pathlib
+from copy import copy
+
+import batoid
+import galsim
+import numpy as np
+import yaml
+from lsst.ts.imsim import OpdMetrology
```

Comment:
If you only needs the `calc_pssn()` from `ts_imsim`, please put it into this `ts_ofc` and let the `ts_imsim` to call it. In the summit deployment, I think we might not deploy the `ts_imsim` (only simulation purpose) to minimize the space.

In addition, the software dependency (or flow) was designed to be `ts_wep` --> `ts_ofc` --> `ts_imsim`. But if you let `ts_ofc` to call the function of `ts_imsim`, you will get the circular dependency, which usually we tried to avoid as much as possible. The circular dependency would give a lot of trouble in the final.

Reply:
I see your point. Moving calc_pssn into ts_ofc would mean moving this "https://github.com/lsst-ts/ts_imsim/blob/develop/python/lsst/ts/imsim/utils/metro_tool.py" all to ts_ofc. I can do that, however I would like to point out a counter-argument. This function will not be used often, that is why is placed in the scripts directory. For the generate_sensitivity_matrix script, for instance, we decided to keep it in ts_ofc even though it imports "batoid_rubin" which is not available in ts_ofc. It is just to have them in the same repository so that if someone needs to regenerate these values, they can. In that sense, given the sporadic use of this script one could argue that it's not really circular. 

But if you have a strong opinion on this, I am happy to move all the metro_tool.py over here, I am just worried it will be a lot of code for a function that we only use once or twice and not in the closed-loop.

Reply:
The cycle dependency would be a disaster for the software maintenance.
Please discuss with UW team to decide to put this `metro_tool.py` to the `ts_wep` or `ts_ofc` for the downstream packages to use (this depends on the `ts_wep` will need it or not).
This can be a new DM ticket.
As a reference, you can see: https://en.wikipedia.org/wiki/Circular_dependency

Although I could understand the point here that this function will be rarely used, I still believe to avoid the cycle dependency is a good software habit, which can decrease the subtle bugs in the future.